README.md

```diff
@@ -1,70 +1,100 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Air Pollution Data App
+
+This is a Single Page Application designed for mobile phones. It is created with React & Redux and uses 2 different API's to store a list of countries, cities and air pollution data for a given city. It has a search bar where you can directly look up for air pollution data of a city.
+
+
+## Built With
+
+- Editor : Visual studio code
+
+- Languages
+  - [x] HTML
+  - [x] CSS
+  - [x] JavaScript
+
+- Libraries
+  - [x] React
+  - [x] Redux
+
+## Live Demo Link
+
+Coming soon...
+
```

Comment:
- [ ] Please endeavour to deploy the application using Heroku, Netlify or GitHub Page. You need to do this in order to get approved 😎. 
- [ ] Once you've deployed the application, kindly add the live link to your README. This is important so that people who are perusing your README and quickly take a look at the live app and see what your project is all about 👍